sui/gateway.js

```diff
@@ -150,11 +151,47 @@ async function callContract(keypair, client, config, chain, args, options) {
 }
 
 async function approveMessages(keypair, client, config, chain, args, options) {
-    if (!chain.contracts.axelar_gateway) {
+    if (!chain.contracts.AxelarGateway) {
         throw new Error('Axelar Gateway package not found.');
     }
 
-    const contractConfig = chain.contracts.axelar_gateway;
+    const contractConfig = chain.contracts.AxelarGateway;
+    const packageId = contractConfig.address;
+    const [multisigSessionId] = args;
+    const { payload, status } = await getMultisigProof(config, chain.axelarId, multisigSessionId);
+
+    if (!payload.messages) {
+        throw new Error('No messages to approve');
+    }
+
+    if (!status.completed) {
+        throw new Error('Multisig session not completed');
+    }
+
+    const executeData = executeDataStruct.parse(new Uint8Array(arrayify('0x' + status.completed.execute_data)));
```

Comment:
I think `Uint8Array` instantiation is unnecessary here because the return type of `arrayify` is `Uint8Array`.
Fix this in the `rotateSigners` function as well.